tests/test_quick_start.py

```diff
@@ -427,11 +484,18 @@ def test_interacting_pretrained_model(self, model, conf_file, model_dir, mode):
     def test_interacting_pretrained_model_api(self, model, conf_file, model_dir, mode):
         if 'IP' in mode:
             self.interact_api(test_configs_path / conf_file)
+        else:
+            pytest.skip("Unsupported mode: {}".format(mode))
+
+    def test_interacting_pretrained_model_socket(self, model, conf_file, model_dir, mode):
+        if 'IP' in mode:
+            for socket_type in ['TCP', 'UNIX']:
```

Comment:
We should think on when and if we want to test for UNIX-sockets, as these tests could fail on Windows